cmd/examples/input/mouse_hitbox/test_scene.go

```diff
@@ -34,7 +35,10 @@ func (scene *TestScene) updateLabel() {
 		return
 	}
 
-	mp := rl.GetMousePosition()
+	var mp rl.Vector2
+	mainthread.Call(func() {
+		mp = rl.GetMousePosition()
+	})
```

Comment:
I think we can get around this by moving the input vector for keyboard/mouse/joystick out of the input system struct and into entities that have `InputState` components, or something like that. It could start out as singleton for keyboard or mouse, or we could adapt it to use multiple gamepads or midi devices even.

Reply:
Woops, I missed this one. This call to raylib isn't necessary in examples anymore, you can just do `scene.Sys.Input.MousePosition` to get the current mouse position, this field gets updated every time the input system ticks.